initfiles/bash/etc/init.d/hpcc-init.in

```diff
@@ -263,6 +267,10 @@ if [ -z $arg ] || [ $# -ne 1 ]; then
     print_usage
 fi
 
+log "Debug log written to $LOG_DIR/hpcc-init.debug"
+exec 2>$LOG_DIR/hpcc-init.debug
+set -x
```

Comment:
@xwang2713 we still collect all of the debugging information.  We stash it in hpcc-init.debug now which sits alongside hpcc-init.log.  So we still have the same level of logging as before, but we have a more user friendly log file for the end user.